infrastructure/deployments/sources-google-webhook.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sources-google-webhook
+  namespace: default
+  labels:
+    app: sources-google-webhook
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      app: sources-google-webhook
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sources-google-webhook
+    spec:
+      containers:
+        - name: app
+          image: ghcr.io/airyhq/sources/google-webhook:latest
+          imagePullPolicy: Always
+          env:
+            - name: KAFKA_BROKERS
+              value: airy-cp-kafka-headless:9092
+            - name: KAFKA_SCHEMA_REGISTRY_URL
+              value: http://airy-cp-schema-registry:8081
+            - name: SERVICE_NAME
+              value: sources-google-webhook
+            - name: GOOGLE_PARTNER_KEY
+              value: "<google_partner_key>"
+          livenessProbe:
+            httpGet:
+              path: /actuator/health
+              port: 8080
+              httpHeaders:
+                - name: Health-Check
+                  value: health-check
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            failureThreshold: 3
+        - name: ngrok
+          command:
+            - /bin/bash
+            - -c
+            - /usr/local/bin/ngrok -config=/etc/ngrok/config.yml -log=stdout -subdomain="<ngrok_client_string>" 8080
+          image: ghcr.io/airyhq/infrastructure/ngrok-client:latest
+          imagePullPolicy: Always
+          resources:
+            limits:
+              cpu: 50m
+              memory: 64Mi
+          volumeMounts:
+            - mountPath: /etc/ngrok
+              name: ngrok-client-config
+      volumes:
+        - name: ngrok-client-config
+          configMap:
+            name: ngrok-client-config
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ngrok-client-config
+  namespace: default
+data:
+  config.yml: |
+    server_addr: proxy.tunnel.airy.co:4443
+    trust_host_root_certs: true
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: public-urls
```

Comment:
This already exists in `sources-facebook-webhook.yaml ` , where the FB_WEBHOOK_PUBLIC_URL is set.

I think would be better to have one configmap which holds all the public urls, outside of both deployment yamls.

Reply:
Ok, thanks. I will get right on it!